pydnd/mechanics/views.py

```diff
@@ -24,3 +25,73 @@ class DamageTypeList(generics.ListCreateAPIView):
 class MagicSchoolList(generics.ListCreateAPIView):
     queryset = MagicSchool.objects.all()
     serializer_class = MagicSchoolListSerializer
+
+class LanguageGet(APIView):
+
+    serializer_class = LanguageListSerializer
+
+    def get(self, request, name_or_id):
+        return get_language(name_or_id)
```

Comment:
I would say ditto to all of these as the one above.

UNLESS we wanted to make some super generic class like `get_model(model_name, name_or_id` and have `model_name.objects.get()` in each. That could work.